src/miner.h

```diff
@@ -14,6 +14,7 @@ class CBlockIndex;
 class CReserveKey;
 class CScript;
 class CWallet;
+namespace Consensus { class Params; };
```

Comment:
Nit: If you mind add a new-line here, but that's not meant to be of any interrest if this is going to get merged now :).